### DISCUSSION TITLE
How to change the order/fields of Many To One relation field

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#3010] [Archive]
When I pick M2O relations I see it like this:

<img width="855" alt="Screen Shot 2020-11-09 at 16 08 04" src="https://user-images.githubusercontent.com/4686040/98551209-b9f93f80-22a5-11eb-9653-fe83a66ce203.png">

Is it possible to change the fields which are showing here? What about simple drop drown, can we configure that as well?

**Comment by benhaynes** [comment]
You can enable/disable fields, as well as reorder them, within Layout options.

![Screen Shot 2020-11-09 at 9 09 43 AM](https://user-images.githubusercontent.com/522079/98551532-679b2d00-226b-11eb-93d0-7c66f51749db.png)

**Reply by botzill** [reply]
The same order that is defined on the table order will be applied here as well? What if we want to make this persistence and any other users that can login will see this order, does that work this way?

**Reply by rijkvanzanten** [reply]
Yeah, you can configure that under Presets in settings 👍
